composer.json

```diff
@@ -18,17 +18,18 @@
         }
     ],
     "require": {
-        "composer-plugin-api": "^1.1 || ^2.0",
-        "php": "^8.0",
+        "composer-plugin-api": "^2.0",
+        "php": "^8.1",
         "leongrdic/smplang": "^1.0.2",
-        "symfony/filesystem": "^5.4 || ^6.0",
-        "symfony/finder": "^4.4.23 || ^5.4 || ^6.0"
+        "symfony/filesystem": "^6.0 || ^7.0",
+        "symfony/finder": "^6.0 || ^7.0"
```

Comment:
<img width="825" alt="Screenshot 2024-03-16 at 17 01 33" src="https://github.com/pact-foundation/composer-downloads-plugin/assets/2605791/95163dab-a71a-4524-bc9a-373698535b1c">

While 5.4 may well still be supported, PACT is now 8.1 only, so this is all it needs. I did have some issues when just using 7.0 and when trying to support all 3. These two do not exhibit the same problems.

Reply:
I wonder what are those issues. I add Symfony 5.4 back and test and look like it works (apart of Windows's rar issue) https://github.com/pact-foundation/composer-downloads-plugin/actions/runs/8309331776/job/22740567129

Reply:
I suggest this change:

```diff
+         "symfony/finder": "^5.4 || ^6.4 || ^7.0"
```

Reply:
> PACT is now 8.1 only, so this is all it needs

But PACT list `tienvx/composer-downloads-plugin` (soon will be `pact-foundation/composer-downloads-plugin`) in `require`, not `require-dev`, and I'm worrying that some projects that are using Symfony 5.4 will not able to install PACT because of dependency conflicts (e.g. those project only require `"symfony/finder: ^5.4"`)